library/src/blas1/rocblas_axpy.cpp

```diff
@@ -3,13 +3,11 @@
  *
  * ************************************************************************ */
 #include <hip/hip_runtime.h>
-
- 
-
 #include "rocblas.h"
- 
 #include "definitions.h"
 
+extern "C" half2 __v_pk_fma_f16(half2, half2, half2) __asm("llvm.fma.v2f16");
```

Comment:
Just to add here too:  We do not want developers seeing this ugliness and copying it into their own code.